crates/bevy_asset/src/lib.rs

```diff
@@ -423,13 +417,13 @@ impl AssetApp for App {
 pub struct TrackAssets;
 
 /// Schedule where [`Assets`] resources are updated.
-#[derive(Debug, Hash, PartialEq, Eq, Clone, ScheduleLabel)]
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
 pub struct UpdateAssets;
 
 /// Schedule where events accumulated in [`Assets`] are applied to the [`AssetEvent`] [`Events`] resource.
 ///
 /// [`Events`]: bevy_ecs::event::Events
-#[derive(Debug, Hash, PartialEq, Eq, Clone, ScheduleLabel)]
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
 pub struct AssetEvents;
```

Comment:
Currently unused, could be used to maybe keep backwards compatibility, but the migration should be very simple
That would mean this has to wait until 0.14 though